public/editor/scripts/main.js

```diff
@@ -79,5 +82,5 @@ require(["jquery", "bowser"], function($, bowser) {
     });
   }
 
-  require(["bramble-editor", "project/project", "sso-override", "fc/project-rename"], init);
+  require(["bramble-editor", "project/project", "sso-override", "fc/project-rename", "reloadProtection"], init);
```

Comment:
this name is the part that matches against what you defined in  public/editor/scripts/main.js  above